common/test/acceptance/pages/studio/index.py

```diff
@@ -287,3 +288,84 @@ class HomePage(DashboardPage):
     Home page for Studio when logged in.
     """
     url = BASE_URL + "/home/"
+
+
+class AccessibilityPage(IndexPage):
+    """
+    Home page for Studio when logged in.
+    """
+    url = BASE_URL + "/accessibility"
+
+    def is_browser_on_page(self):
+        """
+        Is the page header visible?
+        """
+        return self.q(css='#root h1').visible
+
+    def header_text_on_page(self):
+        """
+        Check that the page header has the right text.
+        """
+        return 'Individualized Accessibility Process for Course Creators' in self.q(css='#root h1').text
+
+    def fill_form(self, email, name, message):
+        """
+        Fill the accessibility feedback form out.
+        """
+        email_input = self.q(css='#root input#email')
+        name_input = self.q(css='#root input#fullName')
+        message_input = self.q(css='#root textarea#message')
+
+        email_input.fill(email)
+        name_input.fill(name)
+        message_input.fill(message)
+
+    def submit_form(self):
+        """
+        Click the submit button on the accessibiltiy feedback form.
+        """
+        button = self.q(css='#root section button')[0]
+        button.click()
+        # For some reason, button clicking is flaky :( so click it twice to be sure.
+        button.click()
+        self.wait_for_ajax()
+
+    def leave_field_blank(self, field_id, field_type='input'):
+        """
+        To simulate leaving a field blank, click on the field, then press TAB to move off focus off the field.
+        """
+        field = self.q(css='#root {}#{}'.format(field_type, field_id))[0]
+        field.click()
+        field.send_keys(Keys.TAB)
+
+    def success_alert_shown(self):
```

Comment:
Nit: name would be better as a verb, `wait_for_success_alert` or similar

Reply:
I ended up removing this and `error_alert_shown` since I am now checking if the alert is shown at the end of `submit_form`.